pkgs/misc/emulators/yuzu/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, boost, cmake, git, python2, perl, qt5 , SDL2 }:
+
+stdenv.mkDerivation rec {
+  pname = "yuzu";
+  version = "2019-03-24";
```

```suggestion
  version = "unstable-2019-03-24";
```
https://nixos.org/nixpkgs/manual/#sec-package-naming